mentorship ios/ViewModels/RelationViewModel.swift

```diff
@@ -11,7 +11,7 @@ class RelationViewModel: ObservableObject {
     
     // MARK: - Variables
     @Published var currentRelation = RelationModel().currentRelation
-    @Published var responseData = RelationModel.ResponseData(message: "")
+    @Published var responseData = RelationModel.ResponseData(message: "", success: false)
```

Comment:
is this view model holding all the data in a way it's directly presented without manipulating them?

Reply:
yes. I've now tailored all the view models (till relation view model) and have pushed the commit.